recipe/build.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 if [[ $HOST == *arm64* ]]; then
     autoreconf -i -f
-    (cd wrappers/python && cython --cplus lhapdf.pyx)
+    (cd wrappers/python && PYTHONPATH=$PWD cython --cplus lhapdf.pyx)
```

Comment:
@chrisburr This works, which is great, but can you explain why this is needed?

Reply:
Not fully, maybe it should be `-I` juding by: https://cython.readthedocs.io/en/latest/src/userguide/sharing_declarations.html#search-paths-for-definition-files

My guess is a change in the import behavour in cython 3 but it's not totally clear from their docs. Looking at https://github.com/cython/cython/issues/3862 it seems like it's intentional.

Reply:
> Not fully, maybe it should be `-I` juding by: https://cython.readthedocs.io/en/latest/src/userguide/sharing_declarations.html#search-paths-for-definition-files

Ah nice. I'll get that in my rebase.